### ISSUE TITLE
[BUG] Failed to register layer during installation.

### ISSUE THREAD

**Issue by RiffyDivine** [linuxserver/docker-kasm#50]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Well I am trying to load just ubuntu during the installation and get this error "11d974fc9638 Verifying Checksum 
11d974fc9638 Download complete 
failed to register layer: chtimes /bin: operation not permitted"

Does anyone know what this error means/how to address it?

This is my compose file.
services:
  kasm:
    image: lscr.io/linuxserver/kasm:latest
    container_name: kasm
    privileged: true
    environment:
      - KASM_PORT=4443
      - DOCKER_MTU=1500 #optional
      - NVIDIA_VISIBLE_DEVICES=all
    volumes:
      - /home/riffy/docker/kasm-workspaces/data:/opt
      - /home/riffy/docker/kasm-workspaces/profiles:/profiles #optional
      - /dev/input:/dev/input #optional
      - /run/udev/data:/run/udev/data #optional
    ports:
      - 3000:3000
      - 4443:4443
    restart: unless-stopped
networks: {}

### Expected Behavior

For it to install properly and allow access to the kasm server. 

### Steps To Reproduce

Run the compose file and go to the https://:3000 and during the run of the installer I get the error above and it seems to stop at that point. 

### Environment

```markdown
- OS: Ubuntu
- How docker service was installed: Apt
```

### CPU architecture

x86-64

### Docker creation

```bash
services:
  kasm:
    image: lscr.io/linuxserver/kasm:latest
    container_name: kasm
    privileged: true
    environment:
      - KASM_PORT=4443
      - DOCKER_MTU=1500 #optional
      - NVIDIA_VISIBLE_DEVICES=all
    volumes:
      - /home/riffy/docker/kasm-workspaces/data:/opt
      - /home/riffy/docker/kasm-workspaces/profiles:/profiles #optional
      - /dev/input:/dev/input #optional
      - /run/udev/data:/run/udev/data #optional
    ports:
      - 3000:3000
      - 4443:4443
    restart: unless-stopped
networks: {}
```

### Container logs

```bash
kasm  | [migrations] started
kasm  | [migrations] no migrations found
kasm  | usermod: no changes
kasm  | ───────────────────────────────────────
kasm  | 
kasm  |       ██╗     ███████╗██╗ ██████╗
kasm  |       ██║     ██╔════╝██║██╔═══██╗
kasm  |       ██║     ███████╗██║██║   ██║
kasm  |       ██║     ╚════██║██║██║   ██║
kasm  |       ███████╗███████║██║╚██████╔╝
kasm  |       ╚══════╝╚══════╝╚═╝ ╚═════╝
kasm  | 
kasm  |    Brought to you by linuxserver.io
kasm  | ───────────────────────────────────────
kasm  | 
kasm  | To support LSIO projects visit:
kasm  | https://www.linuxserver.io/donate/
kasm  | 
kasm  | ───────────────────────────────────────
kasm  | GID/UID
kasm  | ───────────────────────────────────────
kasm  | 
kasm  | User UID:    911
kasm  | User GID:    911
kasm  | ───────────────────────────────────────
kasm  | 
kasm  | [custom-init] No custom files found, skipping...
kasm  | [ls.io-init] done.
kasm  | time="2024-03-18T16:27:12.740948501Z" level=error msg="failed to initialize a tracing processor \"otlp\"" error="no OpenTelemetry endpoint: skip plugin"
kasm  | { hostname: 'kasm' }
```

**Comment by thelamer** [comment]
What kind of filesystem is `/home/riffy/docker/kasm-workspaces/data` it needs to be a sane linux filesystem as you are bind mounting in for DinD to function. 

Also did you install docker from here ? https://docs.docker.com/engine/install/ubuntu/

**Comment by RiffyDivine** [comment]
> What kind of filesystem is `/home/riffy/docker/kasm-workspaces/data` it needs to be a sane linux filesystem as you are bind mounting in for DinD to function.
> 
> Also did you install docker from here ? https://docs.docker.com/engine/install/ubuntu/

Yes, I did install it from docker's directions and the mounted folder is a sane address so not sure what you mean by that.

**Comment by thelamer** [comment]
A sane Linux filesystem would be like ext4, btrfs,etc. Is this fuse based in any way ? Or on a remote network share.

**Comment by RiffyDivine** [comment]
> A sane Linux filesystem would be like ext4, btrfs,etc. Is this fuse based in any way ? Or on a remote network share.

No, it's not a file system it's the folder I keep my docker compose files under.

**Comment by thelamer** [comment]
Post output on the host of `mount`
The chtime error usually indicates the filesystem is full, has privilege issues, or is fuse based, which is why I asked.

**Comment by RiffyDivine** [comment]
> Post output on the host of `mount` The chtime error usually indicates the filesystem is full, has privilege issues, or is fuse based, which is why I asked.

I am an idiot is the problem. I just now noticed when looking at it, I was still having it point to a nfs mount and not the hard drive. Changed the docker compose file correctly this time and it's loading. I bet I can guess the issue now looking at it from a step back., it's permissions between the nfs share and the container. Moved all -v from the nfs share to the drive and it works fine. 

I am now getting a /dev/dri/renderD129 error but that likely is my own fault again with how I got the nvidia container toolkit working. I will need to mull that one over tonight or just disable the video card in setup of the container so it doesn't use it.
